app/views/login_logout.py

```diff
@@ -0,0 +1,111 @@
+from flask_restplus import Resource
+from flask import request
+from app.views.user import User
+from app.common.response_genarator import ResponseGenerator
+from app.common.log import logger
+from app import db
+from app import jwt
+from http import HTTPStatus
+from app.common.custom_exception import UserObjectNotFound, PasswordWrong
+from app.models.tokenblocklist import TokenBlockList
+from flask_jwt_extended import create_access_token, get_jwt_identity, jwt_required, get_jwt
+from datetime import datetime, timezone
+import bcrypt
+
+
+class Login(Resource):
+    def post(self):
+        """
+              This is POST API
+             parameters:
+                 email_id: String
+                 password: String
+             responses:
+                 404:
+                     description: User email_id and password not exist
+                 201:
+                     description: Successfully logged in
+                 400:
+                     description: Bad request
+        """
+        try:
+            email_id = request.json.get("email_id", None)
+            password = request.json.get("password", None)
```

Comment:
Why are you setting default value to `None`?
Do not set it to `None` and handle `KeyError`

Reply:
Made the changes as suggested.